File: 1) PYTHON/retangulo.py
import math

altura = float(input( "Digite a altura do retângulo: " ))
base = float(input( "Digite a base do retângulo: " ))

area = altura * base
perimetro = 2 * ( base + altura )
diagonal = math.sqrt( altura ** 2 + base ** 2 )

print( f"""
Área: { area } 
Perímetro: { perimetro }
Diagonal: { diagonal }
""" )

File: 2) JAVA/retangulo.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        int base, altura, perimetro, area;
        double diagonal;
        
        Scanner scanner = new Scanner(System.in);
        
        System.out.println("Digite a base do retangulo: ");
        base = scanner.nextInt();
        
        System.out.println("Digite a altura do retangulo: ");
        altura = scanner.nextInt();
        
        area = base * altura;
        perimetro = 2 * (base + altura);
        diagonal = Math.sqrt((base * base) + (altura * altura));
        
        // Usando printf para uma saída formatada
        System.out.printf("Area: %d, Perimetro: %d, Diagonal: %.2f%n", area, perimetro, diagonal);
        
        scanner.close();
    }
}